server/vitals.js

```diff
@@ -170,10 +207,23 @@ async function checkButtonHeartbeat() {
 
           await db.updateButtonsSentVitalsAlerts(button.id, false)
 
-          // TODO Also send a text message to Responders and Heartbeat Phone Numbers once we know that these messages are reliable
+          // Store the client info
+          if (!clientMessages[client.id]) {
+            clientMessages[client.id] = {
+              client,
+              disconnectedButtons: [],
+              reconnectedButtons: [],
+            }
+          }
+          // Store the reconnected button name
+          clientMessages[client.id].reconnectedButtons.push(button.displayName)
```

Comment:
Cool. The use of a dictionary/object looked weird to me to begin with, but now I see that you needed to add to it from both of these if/else blocks, so totally makes sense.